database/database.go

```diff
@@ -128,6 +129,13 @@ type Database interface {
 	// DeleteRepository deletes repository for the given remote provider's ID.
 	DeleteRepository(remoteID uint64) error
 
+	// GetTokenRecord returns all current update token records.
```

Comment:
Consider if these are necessary if we could avoid the whole `UpdateTokenRecord` type by adding a field to the database?